alacritty_config_derive/src/serde_replace.rs

```diff
@@ -98,9 +98,26 @@ fn match_arms<T>(fields: &Punctuated<Field, T>) -> TokenStream2 {
                 _ => alacritty_config::SerdeReplace::replace(&mut self.#ident, value)?,
             });
         } else {
-            stream.extend(quote! {
-                #literal => alacritty_config::SerdeReplace::replace(&mut self.#ident, next_value)?,
-            });
+            // Check if #[config(alias = "some alias")], and, if present, extract its value
+            let alias_value = field
+                .attrs
+                .iter()
+                .filter_map(|attr| attr.parse_args::<Attr>().ok())
+                .find(|parsed| parsed.ident.as_str() == "alias")
```

Comment:
Find here only allows for a single alias. I think having multiple `#[config(alias = "xxx")]` annotations is technically supported even if it might not be used right now. This should also be taken into account here.

Reply:
Oh, sorry. I was not aware of this. I'll update

Reply:
hello @chrisduerr,
I've just updated according to your request + test to check the multiple `#[config(alias = "some_alias")]` part